docs/en/spark/configuration/sink-plugins/Jdbc.md

```diff
@@ -61,12 +61,15 @@ Configure when `saveMode` is specified as `update` , and when the specified key
 
 Configure when `saveMode` is specified as `update` , whether to show sql
 
+### truncate
+
+Configure when `saveMode` is `overwrite`, whether to truncate table when truncate is true, otherwise to drop table.
+
```

Comment:
@zhaomin1423 I think don't need this description.

Reply:
Thanks, I deleted it.